apps/web/pages/organizations/[organizationId]/invoices/[invoiceId]/components/invoiceDetails.tsx

```diff
@@ -36,72 +36,72 @@ export const InvoiceDetails = ({ invoice, invoiceItems }: InvoiceDetailsProps) =
   const invoiceItemsData = useFragment(InvoiceItemsFragment, invoiceItems)
   return (
     <div className="flex flex-col gap-4 rounded-lg p-4 shadow-md">
-      <div className="flex justify-between">
+      <div className="flex justify-between text-sm">
         <div className="flex flex-col gap-4">
           <div>
             <Image className="m-auto" src="/logo-progwise.svg" alt="Progwise logo" width={60} height={60} />
             <p className="text-lg font-bold">Progwise</p>
-            <p className="text-sm">Greifswald</p>
+            <p>Greifswald</p>
           </div>
           <div>
             <h2 className="text-lg font-bold">Billed to:</h2>
-            <p className="text-sm">{invoiceData.customerName}</p>
-            <p className="text-sm">{invoiceData.customerAddress}</p>
+            <p>{invoiceData.customerName}</p>
+            <p>{invoiceData.customerAddress}</p>
           </div>
         </div>
-        <div className="flex flex-col justify-between gap-4">
+        <div className="flex flex-col justify-between">
           <div className="text-right">
             <button className="btn btn-primary btn-sm print:hidden" onClick={() => print()}>
               <FaPrint />
               Print
             </button>
           </div>
-
+          <div className="flex self-end">
+            <button className="btn btn-primary btn-sm print:hidden">Send</button>
+          </div>
```

Comment:
```suggestion
```

I think it makes more sense to have the send button underneath the table, right now placement is a bit awkward

Reply:
I agree. This is also supposed to be in the other ticket.